### PR TITLE
refactor: Remove unused packages.

### PR DESCRIPTION
**Pull request by saul-jb**
This PR removes packages that are not used and replaces a couple of packages that are only used in one place, helping reduce package size.

- [logplease](https://www.npmjs.com/package/logplease) - Unused. (If a logger is needed I would suggest using [@libp2p/logger ](https://www.npmjs.com/package/@libp2p/logger) to conform the usage to that of Libp2p and IPFS which this package uses.)
- [p-map](https://www.npmjs.com/package/p-map) - Used once and I've [replaced](https://github.com/orbitdb/orbit-db-nextgen/commit/23aee1d1ece7150a3d2c6a7fae63b95a032deb16#diff-f96904fa9ba035fb31cd09aaea921d110674b7d0aa80c18aaf1b9ce04b00a632R298) with `Promise.all`.
- [p-map-series](https://www.npmjs.com/package/p-map-series) - Used once and I've [replaced](https://github.com/orbitdb/orbit-db-nextgen/commit/7a9e002eadc9135e4d5dd962556f9baa8b755146#diff-81dafb74ef70bab09f8587fed64d73df857cf59664241cc91d73518fba439468R106) with `Promise.all`.
- [cpy-cli](https://www.npmjs.com/package/cpy-cli) - Unused.
- [remark-cli](https://www.npmjs.com/package/remark-cli) - Unused.

The last couple of packages are CLI packages that are not used in any command in side`package.json` so if they are being used for development they are probably better being installed globally.

### CODE REVIEW

**Comment by haadcode** [comment]
Leave `p-map` and `p-map-series`, others can go 👍

File: src/oplog/log.js [hunk 1]
```diff
@@ -289,14 +288,15 @@ const Log = async (identity, { logId, logHeads, access, entryStorage, headsStora
...
-          const fetchEntries = async (hash) => {
+          const fetchEntries = (hash) => {
             if (!traversed[hash] && !fetched[hash]) {
               fetched[hash] = true
               return get(hash)
             }
           }
           // Fetch the next/reference entries
-          const nexts = await pMap(toFetch, fetchEntries)
+          const nexts = await Promise.all(toFetch.map(p => Promise.resolve(p).then(fetchEntries)))
```
Comment (haadcode) [inline src/oplog/log.js]: Can this be `const next = await Promise.all(toFetch.map(fetchEntries))`?

Comment (saul-jb) [reply src/oplog/log.js]: Quite right, I thought `toFetch` might contain Promises but it doesn't. Updated.

**Comment by haadcode** [comment]
LGTM, thanks @saul-jb!

**Pull request merged**